packages/vite/src/index.js

```diff
@@ -19,7 +19,7 @@ module.exports = function prefreshPlugin(options = {}) {
           : options && options.ssr === true;
       if (
         shouldSkip ||
-        !/\.(t|j)sx?$/.test(id) ||
+        !/\.(t|j)sx$/.test(id) ||
```

Comment:
This is a bit dangerous for custom-hook files as those aren't jsx but js

Reply:
I was worried about that. A better check is in order then, I'll need to figure that out.